frontend/packages/knative-plugin/src/plugin.tsx

```diff
@@ -174,16 +174,6 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
-  {
```

Comment:
sorry, I didn't get this. Why we are removing extension of type `Page/Resource/Details`, we don't need it.

Reply:
is it because revisions kind is different unlike service and route